src/puja_poreddiwar/PrintDifference.java

```diff
@@ -0,0 +1,60 @@
+package puja_poreddiwar;
+
+public class PrintDifference {
+	int sumOfOdd = 0;
+	int sumOfEven = 0;
+	int temp;
+	void printAllOddDigits(String str) 
+	{
+		System.out.print("sum of all the odd digits in the given String : " );
+		
+		for (int i = 0; i < str.length(); i++) 
+		{
+			char ch = str.charAt(i);
+			boolean flag = Character.isDigit(ch);
+			if (flag == true) 
+			{
+				int num = Character.getNumericValue(ch);
+				if (num % 2 != 0) 
+				{
+					sumOfOdd = sumOfOdd + num;
+				}
+			}
+		}
+		System.out.println(sumOfOdd);
+	}
+	
+	
+	void printAllEvenDigits(String str) 
```

Comment:
Instead of writing 2 methods for even and odd, you can handle this in single method too, try that